.github/workflows/release.yml

```diff
@@ -0,0 +1,22 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
Our tags don't start with v. We should run this on all pushes to master.

Reply:
I don't like the v syntax, there's no need for it. I think just pushes to master is good.

Reply:
As long as merges from develop to master make a release based on that merge commit and not for every commit in develop being merged.

Reply:
Need to test this somehow.

Reply:
Ah ok that would make sense. Should be able to test just running this in your own fork of the repo.